notifyall/notifyall.php

```diff
@@ -12,41 +12,53 @@
 use Friendica\Core\L10n;
 use Friendica\Util\Emailer;
 
-function notifyall_install() {
+function notifyall_install() 
+{
 	logger("installed notifyall");
 }
 
-function notifyall_uninstall() {
+function notifyall_uninstall() 
+{
 	logger("removed notifyall");
 }
 
 function notifyall_module() {}
 
-function notifyall_addon_admin(&$a, &$o) {
-
+function notifyall_addon_admin(&$a, &$o) 
```

Comment:
Can you please include proper type-hint `App` here and remove `&` from `$a`? Or I have to add it to my PR and wait for this being merged.

Reply:
And don't forget `use Friendica\App;` in file header. :-)